index.html

```diff
@@ -256,10 +256,10 @@ <h2>Securing JSON-LD Verifiable Presentations with JOSE</h2>
           <code>application/vp+ld+json</code> with JOSE.
         </p>
         <p>[[rfc7515]] MAY be used to secure this media type.</p>
```

Comment:
```suggestion
        <p>[[rfc7515]] MAY be used to secure this media type. The result of such securing MUST be used as the JWT component of an SD-JWT.</p>
```

Reply:
Same as other comments, I would like to do this in a separate PR.